applications/StructuralMechanicsApplication/custom_elements/truss_elements/linear_truss_element.cpp

```diff
@@ -471,8 +471,13 @@ void LinearTrussElement<TDimension, TNNodes>::CalculateLocalSystem(
         strain_vector[0] = inner_prod(B, nodal_values);
         mConstitutiveLawVector[IP]->CalculateMaterialResponsePK2(cl_values); // fills stress and const. matrix
 
+        double pre_stress = 0.0;
+        if (r_props.Has(TRUSS_PRESTRESS_PK2)) {
+            pre_stress = r_props[TRUSS_PRESTRESS_PK2];
+        }
```

Comment:
This is more a matter of preference, but this could be a one-liner (same in the other places)
```suggestion
        const auto pre_stress = r_props.Has(TRUSS_PRESTRESS_PK2) ? r_props[TRUSS_PRESTRESS_PK2] :  0.0;
```